en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md

```diff
@@ -0,0 +1,139 @@
+# Adding Multiple Keys to the Primary Keystore
+
+WSO2 Identity Server enables adding multiple keys to the primary keystore. Let's explore the following topics to learn more.
+
+!!! tip "Before you begin"
+
+	Sign in to the WSO2 Identity Server Management Console that will be hereafter called Management Console.
+
+
+## Adding new keys to the super tenant
+
+Follow the steps below to add new keys to the super tenant:
+
+1. 	Locate the `wso2carbon.jks` file in the `<IS_HOME>/repository/resources/security` directory in a command prompt. 
+
+	!!! info 
+		wso2carbon is the default keystore. To learn how to change the default keystore see, [Creating New Keystores](../../administer/creating-new-keystores) and [Configuring Keystores](../../administer/configuring-keystores-in-wso2-products).
+
+2.	To generate the new keys and add them to the keystore. 
+
+	1.	Navigate to the `<IS_HOME>/repository/resources/security` directory on a command prompt. 
+
+		!!! info
+			By default, all WSO2 Identity Server keystores are stored here. You may change it based on your requirement.
+
+	2.	Execute the following command.  
+
+		```curl tab="Format"
+		keytool -genkey -alias <PUBLIC_CERTIFICATE_ALIAS> -keyalg RSA -keysize 2048 -keystore <KEYSTORE_NAME> -dname "CN=<<Common Name>>,OU=<<Organization Unit>>,O=<<Organization>>,L=<<Locality>>,S=<<StateofProvice Name>>,C=<<Country Name>>"-storepass <KEYSTORE_PASSWORD> -keypass <PRIVATE_KEY_PASSWORD>
+		```
+
+		``` curl tab="Example"
+		keytool -genkey -alias newkey -keyalg RSA -keysize 2048 -keystore wso2carbon.jks -dname "CN=localhost, OU=IT,O=ABC,L=SL,S=WS,C=LK" -storepass wso2carbon -keypass wso2carbon
+		```
+
+	!!! tip
+		If you are planning to delete the newly-added keys in the future, it is recommended to maintain separate keystores for internal and external encryption purposes.
```

Comment:
Let's rephrase this in a way that indicates that you will remove keys where you are periodically doing key rotation.